### PR TITLE
Added middleware example

### PR DESCRIPTION
**Pull request by Juhlinus**
Thought it might be a good idea to show this example that @iksaku shared on Discord. :smile:

### CODE REVIEW

File: pages/server-side-rendering.mdx [hunk 1]
```diff
@@ -18,3 +18,5 @@ Inertia does not currently support server-side rendering. If that's a priority f
 ## Prerending services
 
 Some Inertia users are successfully using services like [Prerender.io](https://prerender.io) and [Prerender.cloud](https://www.prerender.cloud) to generate and cache HTML versions of their Inertia websites, and then serve that content to bots for SEO purposes.
+
+GitHub user [iksaku](https://gist.github.com/iksaku) has shared a middleware for using Prepender. You can find it [here](https://gist.github.com/iksaku/5efaede73f694942f3ea22af75a18c41).
```
Comment (m1guelpf) [inline pages/server-side-rendering.mdx]:
```suggestion
GitHub user [iksaku](https://gist.github.com/iksaku) has shared a middleware for using [Prerender Cloud](https://www.prerender.cloud/). You can find it [here](https://gist.github.com/iksaku/5efaede73f694942f3ea22af75a18c41).
```

**Comment by reinink** [comment]
Hey @Juhlinus. I don't mean to keep closing your PRs, I really do appreciate your efforts(!), but I just don't think I want to go down this road trying to explain how to do server-side rendering. Further, this is only a Laravel example, and there could be folks using other frameworks (ie. Rails). So, I think I'm going to close this. Hope that makes sense.

**Comment by Juhlinus** [comment]
Haha, it's okay, @reinink !

In my opinion proposing things and getting rejected is worth just as much as getting accepted. It narrows down what - you in this case - are looking for in your project. :smile:

**Comment by reinink** [comment]
Awesome man, great attitude! 💪

**Pull request closed without merging**